index.html

```diff
@@ -0,0 +1,196 @@
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js" integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+  </head>
+
+  <body class="bg-light">
+    <header id="header">
+      <nav class="navbar navbar-expand-lg fixed-top navbar-light bg-light" id="nav-bar">
+        <div class="container">
+          <a class="navbar-brand" href="#">
+                <img src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" class="img-amazon" alt="" id="header-img">
+              </a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+          <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav ml-auto">
+              <li class="nav-item">
+                <a class="nav-link" href="#features">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#works">How it works</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#pricing">Pricing</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <div class="row row-padding">
+      <div class="col-12">
+        <h4 class="alig">
+          <b>Handcrafted, home-made masterpieces</b>
+        </h4>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-4"></div>
+      <div class="col-4">
+        <form id="form" method="POST" action="https://www.freecodecamp.com/email-submit">
+          <div class="form-group">
+            <input type="email" class="form-control" name="email" id="email" placeholder="Enter email" required>
+          </div> <p></p><p></p>
+          <div class="form-group">
+            <div class="align-center">
+              <input type="submit" id="submit" class="btn btn-warning text-center"  value="GET STARTED"/>
+            </div>
+          </div>
+        </form>
+      </div>
+      <div class="col-4"></div>
+    </div> 
+    <div class="row top-padding">
+      <div class="row">
+          <div class="col-4 right-padding">
+            <i class="fas fa-fire ffire"></i>
+
+          </div>
+          <div class="col-7">
+              <div>
+                <h4><b> Premium Materials</b></h4>
```

Comment:
As mentioned earlier, please use CSS to increase the font weight instead of embedding b in the h4 tag

Reply:
This is a very nice and polite review comment 🎉. You have done a good job!